files/zh-tw/web/html/element/wbr/index.md

```diff
@@ -0,0 +1,105 @@
+---
+title: <wbr>：換行機會元素
+slug: Web/HTML/Element/wbr
+---
+
+{{HTMLSidebar}}
+
+**`<wbr>`** [HTML](/zh-TW/docs/Web/HTML) 元素代表換行機會，即文本中的位置，瀏覽器可以選擇在該位置斷行，儘管其斷行規則在該位置本身不會創建斷行。
+
+{{EmbedInteractiveExample("pages/tabbed/wbr.html", "tabbed-shorter")}}
+
+## 屬性
+
+此元素僅包括[全域屬性](/zh-TW/docs/Web/HTML/Global_attributes)。
+
+## 注意事項
+
+在 UTF-8 編碼的頁面上，`<wbr>` 的行為類似於 `U+200B 零寬空格` 代碼點。特別是，它的行為類似於 Unicode 的雙向 BN 代碼點，意味著它對 {{Glossary("bidi")}} 排序沒有影響：`<div dir=rtl>123,<wbr>456</div>` 顯示為 `123,456` 而不是 `456,123`。
+
+出於相同的原因，`<wbr>` 元素在斷行點不引入連字符。要使連字符只出現在行末，請改用軟連字符字符實體（`&shy;`）。
+
+## 範例
+
+*[雅虎風格指南](https://web.archive.org/web/20121014054923/http://styleguide.yahoo.com/)*建議[在標點符號之前斷開 URL](https://web.archive.org/web/20121105171040/http://styleguide.yahoo.com/editing/treat-abbreviations-capitalization-and-titles-consistently/website-names-and-addresses)，以避免在行末留下讓讀者誤以為是 URL 結尾的標點符號。
```

Comment:
```suggestion
[_雅虎風格指南_](https://web.archive.org/web/20121014054923/http://styleguide.yahoo.com/)建議[在標點符號之前斷開 URL](https://web.archive.org/web/20121105171040/http://styleguide.yahoo.com/editing/treat-abbreviations-capitalization-and-titles-consistently/website-names-and-addresses)，以避免在行末留下讓讀者誤以為是 URL 結尾的標點符號。
```

The styling is broken:

![image](https://github.com/mdn/translated-content/assets/15844309/357def9f-e572-4fe6-a3dd-b11e6cfd77cf)